src/base/_typography.scss

```diff
@@ -15,8 +16,8 @@
 
 html {
   @include fluid.font-size(
-    20em,
-    75em,
+    breakpoint.$s,
```

Comment:
Can this be `breakpoint.$xs`? I'd love the type to start sizing upward from `20em` as it did before.